_justifications/example.md.template

```diff
@@ -0,0 +1,40 @@
+---
+title: Justification title
```

Comment:
doesnt a justification have some kind of severity? like WARN, ERROR ?!

Reply:
+1, can be added